package-lock.json

```diff
@@ -1042,6 +1042,21 @@
         }
       }
     },
+    "node_modules/@iconify-json/simple-icons": {
```

Comment:
@FabianWilms why do we need "simple-icons"?

Reply:
instead of adding more and more icons to vitepress they decided to use a library for the social icons. You can see the issue in the changelog.